docker/addons/vault/docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
       - magistrala-vault-volume:/vault/file
       - magistrala-vault-volume:/vault/logs
       - ./config.hcl:/vault/config/config.hcl
-      - ./entrypoint.sh:/entrypoint.sh
+      - ./scripts/entrypoint.sh:/entrypoint.sh
```

Comment:
@JeffMboya This can be in `docker/addons/vault`, becuase it is script for docker , and not for setup